dracut/modules.d/90wwinit/parse-wwinit.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
is it /bin/bash here?

Reply:
The other modules I looked at use `/bin/sh`. I kept this same pattern thinking that a POSIX shell may be all that is available in the initramfs environment.